src/command_modules/azure-cli-batchai/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.1.4
```

Comment:
Put the heading under 0.1.3.
The current released version is 0.1.2 right now so 0.1.3 is fine.
https://pypi.python.org/pypi/azure-cli-batchai

Then you don't need to change setup.py.

Reply:
done